### ISSUE TITLE
quesion : Gumbel-Sigmoid

### ISSUE THREAD

**Issue by adws2** [McGill-NLP/polytropon#9]
Hi, I read your paper with great interest.

However, I don't understand the implementation of the Gumbel-Sigmoid part.

How can Eq.2 in your paper become below?

```python
skill_logits = RelaxedBernoulli(temperature=1., logits=skill_logits).rsample()
```

Also, isn't z_{ij} binary? It doesn't seem to be binary in your code.
